doc/source/appendices/tutorial.rst

```diff
@@ -145,4 +134,20 @@ fluff tracer.  Then update the timeseries plotting script to plot the fluffballs
     git checkout columnphysics/icepack_mechred.F90
     git status
 
-**NOTE:** The file, **doc/source/tutorial/fluff.diff** in the Icepack repository, contains code differences for this example for a version of Icepack from July, 2024.  They may not be relevant for other code versions, but these diffs provide an example of the typical code differences required to add the tracer, fluff.
+Add a New Tracer
+--------------------------------------
+
+In this exercise, add a new tracer associated with fluffballs.
+Call the tracer fluff and make it depend on ice area.  Follow the step-by-step instructions in the :ref:`addtrcr` documentation.
+Once you have implemented the model changes, be sure to add fluffballs output to the standard output diagnostics and turn on the
+fluff tracer.  Then update the timeseries plotting script to plot the fluffballs values over time.
+
+* First, set the initial value, physics, sources, and sinks of fluff to zero and make sure fluff values remain zero throughout the run
+
+* Add some constant atmospheric forcing and review results
+
+* Change the dependency to ice volume, how do the results change?
+
+* Modify the physics to create some physics processes, see isotopes or aerosols for some ideas
+
+**NOTE:** The file, **doc/source/tutorial/fluff.diff** in the Icepack repository, demonstrates code differences for this fluffball activity as implemented in a version of Icepack from July, 2024.  These code difference may not be directly applicable to other code versions, but these diffs provide an example of the typical code differences required to add the tracer, fluff.
```

Comment:
In the main documentation (section 5.5), add a note that there is a tutorial exercise on adding tracers in the appendix?  The tutorial doesn't have a lot of information except for the fluff.diff file, which someone who hasn't looked at the tutorial might find helpful.

Reply:
Done.